discojs/discojs-core/src/dataset/data/image_data.ts

```diff
@@ -18,7 +18,7 @@ export class ImageData extends Data {
     // better error handling. An incorrectly formatted image in the dataset might still
     // cause an error during training, because of the lazy aspect of the dataset; we only
     // verify the first sample.
-    if (!task.trainingInformation.preprocessingFunctions?.includes(ImagePreprocessing.Resize)) {
+    if (task.trainingInformation.preprocessingFunctions?.includes(ImagePreprocessing.Resize) !== true) {
```

Comment:
Why is it a better tested this way?

Reply:
I agree, it's quite weird to test equality with true, hurt me to write it, but eslint doesn't want nullish test, only explicit ones. but as I'm rereading it, it's doesn't improve readability at all.
WDYT of the next one?
```ts
const hasResize = (
  task.trainingInformation.preprocessingFunctions ?? []
).includes(ImagePreprocessing.Resize);
if (!hasResize) { // ..
```

Reply:
I find the last suggestion clearer but I'll let you choose how want to do it!